From de6a0800395f3c7d1f0cf1fbf80393edb1350632 Mon Sep 17 00:00:00 2001
From: Tatsuo ONISHI <onishi@latmos.ipsl.fr>
Date: Thu, 24 Nov 2022 09:14:40 +0100
Subject: [PATCH 45/70] (Update) flags for seas_updates=0 are set explicitly.

TYPE : Update

Description:
       Following setting is added in chem/module_mosaic_addemiss.F.
       Not sure if this is necessary. It's done here just in case
       (i.e. maybe already set somewhere else?)

       for seas_updates = 0,
         icorrection_russell = 0
         icorrection_sst = 0
         icorrection_sofiev_sst = 0
         icorrection_local_organics_Arctic = 0
         icorrection_whitecap = 1

List of modified files:
   * chem/module_mosaic_addemiss.F
---
 chem/module_mosaic_addemiss.F | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/chem/module_mosaic_addemiss.F b/chem/module_mosaic_addemiss.F
index 7c5fa7a..b59d42a 100644
--- a/chem/module_mosaic_addemiss.F
+++ b/chem/module_mosaic_addemiss.F
@@ -887,6 +887,16 @@ size_loop: &
     
 !!!!add the namelist options
 !!!seas_updates = 0 then original code is used; role of xice, Salisbury whitecap and organics only from Fuentes (seas_opt = 3 or 4)!
+!TO2
+       if(config_flags%seas_updates==0) then
+               icorrection_russell = 0
+               icorrection_sst = 0
+               icorrection_sofiev_sst = 0
+               icorrection_local_organics_Arctic = 0
+               icorrection_whitecap = 1
+       endif
+!TO2
+
        if(config_flags%seas_updates==101) then
                icorrection_russell = 1
                icorrection_sst = 1
-- 
2.31.1

